distribution/ddf-common/src/main/resources/etc/log4j2.config.xml

```diff
@@ -104,14 +101,14 @@
 
     <Loggers>
         <Logger name="securityLogger" level="info" additivity="false">
-            <!--<AppenderRef ref="securityFailover"/>-->
+            <AppenderRef ref="securityFailover"/>
             <AppenderRef ref="securityMain"/>
```

Comment:
This line should be removed because `securityMain` is the primary appender of the `securityFailover` Failover.